pycaret/containers/models/classification.py

```diff
@@ -1495,8 +1495,12 @@ def __init__(self, experiment):
         from sklearn.dummy import DummyClassifier
 
         args = {"strategy": "prior", "random_state": experiment.seed}
+
         tune_args = {}
-        tune_grid = {}
+        tune_grid = {
```

Comment:
@Yard1 I think you were going to make this change also. Please check if this matches your change. Also, similar change needs to be made for regression as well.